### PR TITLE
Fix more FP in testing

### PR DESCRIPTION
**Pull request by nasbench**
This PR aims to fix more false positives found during testing

### CODE REVIEW

**Comment by phantinuss** [comment]
Please ping me for #3518 after this PR is merged.

File: rules/windows/image_load/image_load_side_load_antivirus.yml [hunk 1]
```diff
@@ -42,5 +42,7 @@ detection:
             - 'C:\Program Files (x86)\McAfee\'
     condition: (selection_bitdefender and not filter_bitdefender) or (selection_fsecure and not filter_fsecure) or (selection_mcafee and not filter_mcafee)
 falsepositives:
-    - Unknown
+    - Applications that load the same dlls mentioned in the detection section. Investigate them and filter them out if a lot FPs are caused.
+    - Dell SARemediation pulgin folder (C:\Program Files\Dell\SARemediation\plugin\log.dll) is known to contain the 'log.dll' file.
```
Comment (phantinuss) [inline rules/windows/image_load/image_load_side_load_antivirus.yml:46]: typo: pulgin

Comment (phantinuss) [reply rules/windows/image_load/image_load_side_load_antivirus.yml:46]: In general: If you know of specific FPs: Best to filter them using a filter selecection

File: rules/windows/image_load/image_load_susp_python_image_load.yml [hunk 2]
```diff
@@ -6,26 +6,27 @@ date: 2020/05/03
...
+            - 'C:\Program Files (x86)\'
+    condition: selection and not filter
 fields:
-  - Description
+    - Description
 falsepositives:
-  - Legit Py2Exe Binaries
-level: medium
\ No newline at end of file
+    - Legit Py2Exe Binaries
```
Comment (phantinuss) [inline rules/windows/image_load/image_load_susp_python_image_load.yml:30]: Legitimate

File: rules/windows/image_load/image_load_susp_python_image_load.yml [hunk 3]
```diff
@@ -6,26 +6,27 @@ date: 2020/05/03
...
+    - Known false positive caused with anaconda installation/update
```
Comment (phantinuss) [inline rules/windows/image_load/image_load_susp_python_image_load.yml:31]: Can you add a filter selection for that?

File: rules/windows/process_creation/proc_creation_win_mstsc.yml [hunk 4]
```diff
@@ -26,10 +26,13 @@ detection:
             - ' /u'
             - ' /p'
     filter_mstsc_1:
+        # Example: mstsc.exe /v:XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX /hvsocketserviceid:XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX /silent /wslg /plugin:WSLDVC /wslgsharedmemorypath:WSL\XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX\wslg C:\ProgramData\Microsoft\WSL\wslg.rdp
         ParentImage: 'C:\Windows\System32\lxss\wslhost.exe'
+        CommandLine|contains: 'C:\ProgramData\Microsoft\WSL\wslg.rdp'
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_mstsc.yml:31]: Just to be sure: The "and" condition is intended?

File: rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml [hunk 5]
```diff
@@ -27,7 +27,8 @@ fields:
     - CommandLine
     - ParentCommandLine
 falsepositives:
-    - Unknown
+    - During Anaconda update the 'conda.exe' process will eventually launch the command described in the detection section
+    - Other currently unknown cases
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml:31]: we don't need to explain that there are unknown unknowns, I think

File: rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml [hunk 6]
```diff
@@ -27,7 +27,8 @@ fields:
     - CommandLine
     - ParentCommandLine
 falsepositives:
-    - Unknown
+    - During Anaconda update the 'conda.exe' process will eventually launch the command described in the detection section
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml:30]: Can you describe the FP as a filter selection?

File: rules/windows/process_creation/proc_creation_win_mstsc.yml (hunk 4 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_mstsc.yml:31]: Yes it is

File: rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml (hunk 5 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml:31]: Agreed

File: rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml (hunk 6 above)
Comment (nasbench) [reply rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml:30]: I'll add a filter but I would have to rely on the grandparent field which is not available in sysmon EID 1 and 4688. (I'll simply add a note that this won't work without aurora, similar to how did in other rules)

Comment (phantinuss) [reply rules/windows/process_creation/proc_creation_win_susp_codepage_lookup.yml:33]: ok that's suboptimal but if we cannot filter differently, it is as it is.

**Pull request merged**